pkg/cmd/cli/describe/printer.go

```diff
@@ -80,7 +82,41 @@ func NewHumanReadablePrinter(noHeaders bool) *kctl.HumanReadablePrinter {
 	return p
 }
 
-func printParameters(t *templateapi.Template, w io.Writer) error {
+const templateDescriptionLen = 80
+
+func printTemplate(t *templateapi.Template, w io.Writer) error {
+	description := ""
+	if t.Annotations != nil {
+		description = t.Annotations["description"]
+	}
+	if len(description) > templateDescriptionLen {
+		description = strings.TrimSpace(description[:templateDescriptionLen-3]) + "..."
+	}
+	empty, generated, total := 0, 0, len(t.Parameters)
+	for _, p := range t.Parameters {
+		if len(p.Value) > 0 {
+			continue
+		}
+		if len(p.Generate) > 0 {
+			generated++
+			continue
+		}
+		empty++
+	}
+	params := ""
+	switch {
+	case empty > 0:
+		params = fmt.Sprintf("%d blank", empty)
+	case generated > 0:
+		params = fmt.Sprintf("%d generated", generated)
+	default:
+		params = fmt.Sprintf("%d set", total)
```

Comment:
I guess i'm not clear what you're trying to print out here.  I assumed it was "number of parameters with a set value, number with a generated value, and number that are empty(not explicitly set or generated)" but if this is doing that, i don't understand how.